### PR TITLE
Fix/sniff ecms admin templates

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java [hunk 1]
```diff
@@ -911,13 +911,27 @@ public void deleteDocumentTemplate(String title) {
...
+                nameTemplate)))
+                .is(Condition.not(Condition.visible))) {
+          $(ELEMENT_ICON_NEXT_ARROW).click();
+        }
+        if ($(byXpath(ELEMENT_ECM_TEMPLATES_LIST_CHECK_LIST.replace("{$name}", name).replace("{$template}",
+                nameTemplate)))
+                .waitUntil(Condition.visible,Configuration.timeout).isDisplayed()) {
+          break;
+        }
+      }
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: What is the purpose of this loop ? Why 10 iterations ?
Also, the indentation is not good.

Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: the purpose of this loop is to check if the template is added, if the template is not found on the current page,  click on next page, until finding the template

Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: For the indentation, code formatted .

Comment (thomasdelhomenie) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: > the purpose of this loop is to check if the template is added, if the template is not found on the current page, click on next page, until finding the template

OK, so please use a constant for the value with a meaningful name, and add a comment to explain that.

Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: done (y)

**Pull request merged**